packages/indexer-agent/src/commands/start.ts

```diff
@@ -421,6 +421,18 @@ export default {
         default: 'auto',
         group: 'Indexer Infrastructure',
       })
+      .option('auto-min-batch-size', {
+        description: `Minimum number of allocation transactions inside a batch for AUTO allocation management. No obvious upperbound, with default of 1`,
+        type: 'number',
+        default: 1,
+        group: 'Indexer Infrastructure',
+      })
+      .option('auto-max-wait-time', {
```

Comment:
Maybe if this isn't defined by the user it is infinite or 0? So in that case approved actions will auto execute only if the batch size meets the `auto-min-batch-size`?

Reply:
Do you think the name is immediately clear? `auto-min-batch-size` 

I'm not sure that tells a user what kind of batch size it is (we have many batching mechanisms 😄  ). 

What about `auto-allocation-min-batch-size` ?

Reply:
very true, concise names are hard to think of sometimes!

Reply:
Definitely!